internal/extgen/gofile.go

```diff
@@ -121,7 +125,7 @@ func (gg *GoFileGenerator) generateMethodWrapper(method phpClassMethod, class ph
 	builder.WriteString("	obj := getGoObject(handle)\n")
 	builder.WriteString("	if obj == nil {\n")
 	if method.ReturnType != "void" {
-		if method.ReturnType == "string" {
+		if method.ReturnType == "string" || method.ReturnType == "array" {
```

Comment:
Can't we move this logic to the template itself?

Reply:
Moved in the template indeed